src/pages/docs/creators/javascript/guides/providers-filesystem.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Golem concepts: filesystem on remote node'
+description: Explore topics related to file system on a VM run on a remote node to better design your custom image.
+pageTitle: VM Filesystem - how the image design impacts my options.
+type: Guide
+---
+
+# Filesystem on VM run on a remote node.
+
+{% alert level="info" %}
+
+This example utilized the `golem/examples-outbound:latest` image deployed on one of the nodes in the `testnet`. Actual numbers will depend on the node on which you run such a task.
+
+{% /alert %}
+
+Your options of what you can do on a remote node will depend on how your image is configured. You already know that transferring data to and from a provider is possible only to/from folders that are configured as `VOLUME` in your image definition. (See [here](/docs/creators/javascript/guides/golem-images) for information about images and how to create one.)
+
+If you would run `df -h` on a VM build from this image on a provider, you might receive an output like this:
+
+```
+Filesystem Size Used Available Use% Mounted on
+overlay 128.0M 8.0K 128.0M 0% /
+devtmpfs 630.2M 0 630.2M 0% /dev
+tmpfs 632.8M 0 632.8M 0% /tmp
+tmpfs 632.8M 0 632.8M 0% /dev/shm
+mnt0 24.0G 10.2G 13.8G 42% /golem/work
+```
+
+Take a look at this output to better understand some Golem concepts.
+
+- The ‘/’ folder is limited to 128 MB.
+- ‘Tmpfs` folders by default are limited to 50% of the provider's memory available for your payload.
+- The `/` and `tmpfs` folders are stored in the provider's memory.
+- The ‘/golem/work’ folder has much more space available, as it has access to disk storage offered by the provider. While this might not be obvious, this folder was defined as the `VOLUME` directory in the Dockerfile, that was used to create the `golem/examples-outbound: latest` image. On this provider, the user can utilize up to 13.8 GB of disk storage.
```

Comment:
```suggestion
- The ‘/golem/work’ directory has much more space available, as it has access to disk storage offered by the provider. While this might not be obvious, this folder was defined as the `VOLUME` directory in the Dockerfile, that was used to create the `golem/examples-outbound: latest` image. On this provider, the user can utilize up to 13.8 GB of disk storage.
```